docs/topics/http/shortcuts.txt

```diff
@@ -158,6 +178,14 @@ will be returned::
         obj = MyModel.objects.get(...)
         return redirect(obj, permanent=True)
 
+Additionally, the ``preserve_request_method`` argument can be used to preserve
+the original HTTP method::
+
+    def my_view(request):
+        ...
+        obj = MyModel.objects.get(...)
+        return redirect(obj, preserve_request_method=True)
```

Comment:
I think this example should be grown to use a POST request, which is really where the 307/308 status shine. I'll add something to this effect.